dev/tools/vitool/test/vitool_test.dart

```diff
@@ -0,0 +1,667 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
Looks like the failure paths for d="" and transform="" aren't tested here.

Reply:
I'm not a specs pro but if I understand correctly what I read here:
https://www.w3.org/TR/SVG/coords.html#TransformAttribute

transform-list:
    wsp* transforms? wsp*

the question mark after transform means an empty transform is legal? (same for d)

Reply:
i meant that you didn't test things like `d="(*#^%"` and `transform="bad(fu())"`

Reply:
Added tests for illegal transform in last round